subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleVersionsPluginSmokeTest.groovy

```diff
@@ -16,11 +16,15 @@
 
 package org.gradle.smoketests
 
+import org.gradle.util.internal.VersionNumber
+import org.junit.Assume
+
 import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 
 class GradleVersionsPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
 
     def 'can check for updated versions'() {
+        Assume.assumeTrue('Incompatible with Groovy 4: Execution failed for task \':dependencyUpdates\'. > groovy/util/XmlSlurper', VersionNumber.parse(GroovySystem.version).major < 4) // TODO Watch for merge and release of https://github.com/ben-manes/gradle-versions-plugin/pull/656
```

Comment:
Perhaps we should comment on this PR? It seems dead.

Reply:
[Done](https://github.com/ben-manes/gradle-versions-plugin/pull/656#issuecomment-1246089846)